code/modules/spacepod/spacepod.dm

```diff
@@ -0,0 +1,722 @@
+#define DAMAGE			1
+#define FIRE			2
+#define LIGHT			1
+#define WINDOW			2
+#define RIM	    		3
+#define PAINT			4
+
+/obj/item/pod_paint_bucket
+	name = "space pod paintkit"
+	desc = "Pimp your ride"
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "paint_red"
+
+/obj/vehicle/spacepod
```

Comment:
holy hell did you at any point read the vehicle code or what? you're not supposed to inherit from vehicle you're supposed to inherit from vehicle/sealed or vehicle/ridden, (vehicle/sealed in this case). Worst part of this is that you're not even putting this in the vehicles module folder, and instead make a new module folder for some reason.

Reply:
It may also be suggested to make them a subtype of mechs. (Poor MRTY)

Reply:
they'd override every proc in sealed anyways, so it's moot point. Also it's not in vehicles module folder because finished spacepods will have easily 5-10 spacepod files

Reply:
then make a spacepods folder in the vehicles folder..?

the entire purpose of the vehicle/sealed is to override the procs, thats how inheritance works, it's literaly idiotic to have 2 subclasses with the same functionality.

Reply:
turns out vehiclecode is still ass anyways

> \<Qustinnus\> ill eat someones ass before i touch vehicle code again